docs/source/user-guide/introduction.md

```diff
@@ -96,6 +96,7 @@ Here are some active projects using DataFusion:
 
 - [Arroyo](https://github.com/ArroyoSystems/arroyo) Distributed stream processing engine in Rust
 - [Ballista](https://github.com/apache/datafusion-ballista) Distributed SQL Query Engine
+- [Blaze](https://github.com/kwai/blaze) The Blaze accelerator for Apache Spark leverages native vectorized execution to accelerate query processing
```

Comment:
There is an existing entry for `blaze` on line 128 below this. The repo is different though. 

Is this the same project or a different one?

If this is the same blaze project, can you please remove the duplicate listing below?

Reply:
Thanks for your reminding, I have removed 'blaze' from  less active project list.